fastecdsa/curve.py

```diff
@@ -75,7 +75,7 @@ def is_point_on_curve(self, point: (int, int)) -> bool:
         return (left - right) % self.p == 0
 
     def evaluate(self, x: int) -> int:
-        """ Evaluate the elliptic curve polynomial at 'x'
+        r""" Evaluate the elliptic curve polynomial at 'x'
```

Comment:
Note that the `\b` in `\bmod` below is not an *invalid* escape character, it's a backspace. I assumed this was not the intention.